src/pytest_cov/plugin.py

```diff
@@ -226,6 +244,13 @@ def pytest_runtestloop(self, session):
                 compat_session.testsfailed += 1
 
     def pytest_terminal_summary(self, terminalreporter):
+        if self._disabled:
+            markup = {'red': True, 'bold': True}
+            msg = (
+                'WARNING: Coverage disabled by user!'
```

Comment:
I would send that message both to the warner and terminalwriter.

To test the warning you need to configure pytest to show warnings (add `-rw` to commandline).